lib/eslint.js

```diff
@@ -0,0 +1,29 @@
+const CLIEngine = require("eslint").CLIEngine;
+const globby = require("globby");
+const settings = require('./settings');
+
+const getPathPattern = async () => {
+    const pathPattern = [`${settings.paths.bundlesDir}/**/*.js`];
+    const pathPatternEco = `${settings.paths.ecoDir}/**/*.js`;
+    const ecoFilesToVerify = await globby([pathPatternEco]);
```

Comment:
We do we need to verify it on this level?

Reply:
Yes, it's needed. ESlint crashes if gets undefined path to linting and in this case we should to check existing eco modules before.